blog/2022-06-30-luos-and-esp32.mdx

```diff
@@ -0,0 +1,179 @@
+- myProject/
+             - CMakeLists.txt
+             - sdkconfig
+             - components/ - component1/ - CMakeLists.txt
+                                         - Kconfig
+                                         - src1.c
+                           - component2/ - CMakeLists.txt
+                                         - Kconfig
+                                         - src1.c
+                                         - include/ - component2.h
+             - main/       - CMakeLists.txt
+                           - src1.c
+                           - src2.c
+
+             - build/
+```
+
+<h2>Luos Concept</h2>
+
+<div align="center">
+  <Image
+    src="/img/feature-service-package-node-board.png"
+    darkSrc="/img/luos-layers-white.png" alt="Luos Concept explaination"
+  />
+</div>
+
+1. Product. A node or a set of nodes that communicate with each other. This is where you will have the configurations and information intended for all your boards.
+2. Product config. This is a config file for your entire Product (product_config.h)
+3. Node. This is the board's code. This is your actual Eclipse, PlatformIO, IAR, or other projects.
+4. Node config. This is the config file for your node! configure here specific need for the node like pinout or hardware peripheric in HAL (node_config.h)
+5. Package. A shareable folder containing code files managing one or more services.
+6. Service. Luos services can be drivers or apps. Each service can follow Luos profiles to be standard, but it can be custom.
+
+```AsciiDoc
+ Project
+  │
+  ├─── product_config.h
+  └─── Nodes
+        ├─── Node_1
+        │     ├─── node_config.h
+        │     ├─── main.c
+        │     └─── Packages
+        │             ├─── Package_1
+        │             │     ├─── package_1.c
+        │             │     └─── package_1.h
+        │             └─── Package_2
+        │                   ├─── package_2.c
+        │                   └─── package_2.h
+        ├─── Node_2
+        │     ├─── node_config.h
+        │     ├─── main.c
+        │     └─── Packages
+        │             └─── Package_3
+        │                   ├─── package_3.c
+        │                   └─── package_3.h
+        │
+        └─── Node_3
+                ├─── node_config.h
+                ├─── main.c
+                └─── Packages
+                        ├─── Package_4
+                        │     ├─── package_4.c
+                        │     └─── package_4.h
+                        ├─── Package_5
+                        │     ├─── package_5.c
+                        │     └─── package_5.h
+                        └─── Package_6
+                            ├─── package_6.c
+                            └─── package_6.h
+
+```
+
+<h2>Conclusion</h2>
+
+As we can see, Luos and Espressif have many concepts in common! 
+
+ESP32 target is a HAL compatibility for Luos. A project and its configuration can be linked to the product and node with their structure. 
```

Comment:
The end of sentence "to the product and node with their structure." is unclear, we don't understand if "node" is used as a verb or as noun, and what structure we are talking about. Consider rephrasing

Reply:
Modified to: "ESP32 target is a HAL compatibility for Luos. A project and its configuration can be linked to a product composed of nodes with the Espressif structure. "